src/frontend/src/pages/landingpage/faq/faqlist.tsx

```diff
@@ -0,0 +1,30 @@
+import type { PropsWithChildren } from "react";
+
+
+const Link = ({href, children}: PropsWithChildren<{ href: string }>) => <a className="text-blue-700 dark:text-blue-400 hover:underline" href={href}>{children}</a>;
+
+
+export const FAQList: Array<{question: string, answer: string | JSX.Element}> = [
+    {
+        question: "What is the difference to other git analysis sites?",
+        answer: "We offer analysis over all your repositories and not just one repository at a time",
+    },
+    {
+        question: "How secure is this site?",
+        answer: "We use OAuth for your login. Also our cookies and securely encrypted",
```

Comment:
```suggestion
        answer: "We use OAuth 2.0 to authenticate a user, saving you the hassle of creating a new account and a new vulnerability. Furthermore, Gitalytics only has read-only access to your GitHub/GitLab/BitBucket repositories.",
```
"It's secure. Trust me bro."